fe/fe-core/src/main/java/com/starrocks/planner/FragmentNormalizer.java

```diff
@@ -174,10 +202,16 @@ public Integer remapSlotId(Integer slotId) {
     }
 
     public List<Integer> remapSlotIds(List<SlotId> slotIds) {
+        if (slotIds == null || slotIds.isEmpty()) {
```

Comment:
why slotIds may be null ?

Reply:
cheat the compiler and make it not to panic.